utils/tinystr/src/ascii.rs

```diff
@@ -41,6 +41,8 @@ impl<const N: usize> TinyAsciiStr<N> {
         let mut out = [0; N];
         let mut i = 0;
         let mut found_null = false;
+        // Indexing is protected by TinyStrError::TooLarge
```

Comment:
issue: This is work from #1677.

suggestion: This is already merged into main, and you should be able to merge in the latest, per Shane's suggestion.